src/account/account.ts

```diff
@@ -1,13 +1,10 @@
 import algosdk from 'algosdk'
```

Comment:
Every function in this (account) folder is now marked deprecated and the implementation points to the new place.